packages/manager/modules/manager-pci-common/src/components/quantity-selector/QuantitySelector.component.tsx

```diff
@@ -48,116 +35,68 @@ export function QuantitySelector({
-            </OsdsPopover>
+            <>
+              <div id="popover-trigger">
+                <OdsIcon
+                  name={ODS_ICON_NAME.question}
+                  className="cursor-help text-[16px]"
+                  color={ODS_THEME_COLOR_INTENT.text}
+                />
+              </div>
+              <OdsPopover triggerId="popover-trigger" className="w-4 h-4">
+                {labelHelpText}
+              </OdsPopover>
+            </>
           )}
         </div>
 
         {description && (
-          <OsdsText
-            color={ODS_THEME_COLOR_INTENT.text}
-            size={ODS_THEME_TYPOGRAPHY_SIZE._400}
-            className="mb-6"
-          >
+          <OdsText preset="span" className="mb-6">
             {description}
-          </OsdsText>
+          </OdsText>
         )}
-        <OsdsQuantity>
-          <OsdsButton
-            slot="minus"
-            variant={ODS_BUTTON_VARIANT.flat}
-            color={ODS_THEME_COLOR_INTENT.primary}
-            size={ODS_BUTTON_SIZE.sm}
-            data-testid="quantity-button-minus"
-            text-align="center"
-            disabled={min !== undefined && value - 1 < min ? true : undefined}
-          >
-            <OsdsIcon
-              name={ODS_ICON_NAME.MINUS}
-              size={ODS_ICON_SIZE.sm}
-              className="bg-white"
-            />
-          </OsdsButton>
-          <OsdsInput
-            type={ODS_INPUT_TYPE.number}
-            value={value}
-            color={
-              isValid
-                ? ODS_THEME_COLOR_INTENT.primary
-                : ODS_THEME_COLOR_INTENT.error
-            }
-            onOdsValueChange={(event) => {
-              forceRedraw();
-              onValueChange(Number(event.detail.value));
-            }}
-            min={min}
-            max={max}
-          />
-          <OsdsButton
-            slot="plus"
-            variant={ODS_BUTTON_VARIANT.flat}
-            color={ODS_THEME_COLOR_INTENT.primary}
-            size={ODS_BUTTON_SIZE.sm}
-            data-testid="quantity-button-plus"
-            text-align="center"
-            disabled={max !== undefined && value + 1 > max ? true : undefined}
-          >
-            <OsdsIcon
-              className="bg-white"
-              name={ODS_ICON_NAME.PLUS}
-              size={ODS_ICON_SIZE.xs}
-            />
-          </OsdsButton>
-        </OsdsQuantity>
+        <OdsQuantity
+          data-testid="quantity-button-minus"
+          onOdsChange={(event: OdsInputChangeEvent) =>
+            onValueChange(Number(event.detail.value))
+          }
+          min={min}
+          max={max}
+          name="quantity"
+          isDisabled={min !== undefined && value - 1 < min ? true : undefined}
+        />
       </div>
       {error.min && (
-        <OsdsText color={ODS_THEME_COLOR_INTENT.error} slot="helper">
+        <OdsText color={ODS_THEME_COLOR_INTENT.error} slot="helper">
```

Comment:
`ODS_THEME_COLOR_INTENT` and `slot `does not exist anymore
You should use a custom tailwind class: 
```suggestion
        <OdsText className="text-[--ods-color-critical-500]>
```